src/manual/multiple-peerconnections/index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+  <title>WebRTC Test page</title>
+  <script type="text/javascript"
+src="../../../node_modules/webrtc-adapter/out/adapter.js"></script>
+  <script type="text/javascript">
```

Comment:
Please put the javascript in a separate JS file, general rule is that when there is more than a few lines, put it in a separate file. Example: https://github.com/webrtc/samples/tree/gh-pages/src/content/peerconnection/pc1